Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1

```diff
@@ -289,6 +289,23 @@ class DemistoObject {
                 "primaryTerm" = -1
             } } })
     }
+
+    [Object[]] GetIntegrationContextVersioned ($Refresh) {
+        if ( -not $this.IsIntegration ) {
+            throw "Method not supported"
+        }
+        $integration_context = $this.ServerRequest(@{type = "executeCommand"; command = "getIntegrationContext"; args = @{ refresh = $Refresh } })
+
+        return $integration_context.context
```

Comment:
We should be returning also the version. Otherwise I don't see how an integration can use the version.